packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -98,12 +96,12 @@ export class EthChain implements IChain {
         return cb(err);
       }
       // getPendingTxs returns all txps when given a native currency
-      server.getPendingTxs(opts, (err, txps) => {
+      server.getPendingTxs(opts, (err, _txps) => {
         if (err) return cb(err);
         let fees = 0;
         let amounts = 0;
 
-        txps = txps.filter(txp => {
+        _txps.filter(txp => {
```

Comment:
This doesn't look right. This is not effectively filtering, it's just a loop that's summing `fees` and `amounts`. I think we should make it a loop and remove the `_` prefix from txps.